routing/missioncontrol.go

```diff
@@ -215,8 +215,14 @@ func (m *missionControl) NewPaymentSession(routeHints [][]HopHint,
 				TimeLockDelta: hopHint.CLTVExpiryDelta,
 			}
 
-			v := NewVertex(hopHint.NodeID)
-			edges[v] = append(edges[v], edge)
+			from := NewVertex(hopHint.NodeID)
+			to := Vertex(endNode.PubKeyBytes)
+
+			fromNode := &channeldb.LightningNode{PubKeyBytes: from}
+			edges[to] = append(edges[to], &edgePolicyWithSource{
+				sourceNode: fromNode,
```

Comment:
Might it be worth making `soureNode` a `[33]byte` value directly? We waste a lot of space allocating `channeldb.LightningNode`, to only use one field

First commit message appears to be clipped

Reply:
any updates on this?